src/utils/debounce.js

```diff
@@ -0,0 +1,18 @@
+/**
+ *
+ *
+ * @param {*} func - The function to be called by debounce function
+ * @param {*} delay - The time at which the call will be delayed for
+ * @returns {function}
+ */
+const debounceFn = (func, delay) => {
```

Comment:
#### Comment
- What is this for?

Reply:
The debounce function for the bookmarks feature. I decided to move it to another file so it can be reused.